watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab.py

```diff
@@ -0,0 +1,628 @@
+            "process_subtype": "methane",
+        },
+    )
+    m.fs.product_hydrogen = Product(default={"property_package": m.fs.prop})
+    m.fs.product_methane = Product(default={"property_package": m.fs.prop})
+    m.fs.product_H2O = Product(default={"property_package": m.fs.prop})
+
+    # connections
+    m.fs.s01 = Arc(source=m.fs.feed.outlet, destination=m.fs.metab_hydrogen.inlet)
+    m.fs.s02 = Arc(
+        source=m.fs.metab_hydrogen.treated, destination=m.fs.metab_methane.inlet
+    )
+    m.fs.s03 = Arc(
+        source=m.fs.metab_hydrogen.byproduct, destination=m.fs.product_hydrogen.inlet
+    )
+    m.fs.s04 = Arc(
+        source=m.fs.metab_methane.byproduct, destination=m.fs.product_methane.inlet
+    )
+    m.fs.s05 = Arc(
+        source=m.fs.metab_methane.treated, destination=m.fs.product_H2O.inlet
+    )
+    TransformationFactory("network.expand_arcs").apply_to(m)
+
+    # scaling
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def set_operating_conditions(m):
+    # ---specifications---
+    # feed
+    flow_vol = 3.286e-4 * pyunits.m**3 / pyunits.s
+    conc_mass_cod = 6.76 * pyunits.kg / pyunits.m**3
+
+    m.fs.feed.flow_vol[0].fix(flow_vol)
+    m.fs.feed.conc_mass_comp[0, "cod"].fix(conc_mass_cod)
+    m.fs.feed.properties[0].flow_mass_comp["hydrogen"].fix(1e-8)
+    m.fs.feed.properties[0].flow_mass_comp["methane"].fix(1e-8)
+    solve(m.fs.feed)
+
+    # metab_hydrogen
+    m.fs.metab_hydrogen.load_parameters_from_database(use_default_removal=True)
+
+    # metab_methane
+    m.fs.metab_methane.load_parameters_from_database(use_default_removal=True)
+
+
+def initialize_system(m):
+    seq = SequentialDecomposition()
+    seq.options.tear_set = []
+    seq.options.iterLim = 1
+    seq.run(m, lambda u: u.initialize())
+
+
+def solve(blk, solver=None, tee=False, check_termination=True):
+    if solver is None:
+        solver = get_solver()
+    results = solver.solve(blk, tee=tee)
+    if check_termination:
+        assert_optimal_termination(results)
+    return results
+
+
+def display_results(m):
+    unit_list = ["feed", "metab_hydrogen", "metab_methane"]
+    for u in unit_list:
+        m.fs.component(u).report()
+
+
+def add_costing(m):
+    source_file = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        "metab_global_costing.yaml",
+    )
+    m.fs.costing = ZeroOrderCosting(default={"case_study_definition": source_file})
+    # typing aid
+    costing_kwargs = {"default": {"flowsheet_costing_block": m.fs.costing}}
+    m.fs.metab_hydrogen.costing = UnitModelCostingBlock(**costing_kwargs)
+    m.fs.metab_methane.costing = UnitModelCostingBlock(**costing_kwargs)
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_electricity_intensity(m.fs.product_H2O.properties[0].flow_vol)
+
+    # m.fs.costing.add_LCOW(m.fs.product_H2O.properties[0].flow_vol)
+
+    # other levelized costs
+    # TODO -resolve discrepancy between sum of cost components and total levelized costs
```

Comment:
Another TODO - any actions needed?

Reply:
I just added this a few minutes ago. It's something that will have to be revisited.

Reply:
Revisited before this PR merges? Or revisited somewhere down the line?

Reply:
Somewhere down the line.